src/mito/src/engine.rs

```diff
@@ -404,7 +404,8 @@ impl<S: StorageEngine> MitoEngineInner<S> {
 
             let table_id = request.table_id;
             // TODO(dennis): supports multi regions;
-            let region_number = 0;
+            assert_eq!(request.region_numbers.len(), 1);
```

Comment:
```rust
assert_eq!(request.region_numbers, vec![0]);
```

Reply:
`region_number` is assigned by Meta, might not necessarily be "0" here.